host/config/chain_spec_list_default.json

```diff
@@ -82,7 +90,11 @@
         "l2_contract": "0x1670090000000000000000000000000000010001",
         "rpc": "https://rpc.hekla.taiko.xyz",
         "beacon_rpc": null,
-        "sgx_verifier_address": "0x532efbf6d62720d0b2a2bb9d11066e8588cae6d9",
+        "verifier_address":{
+            "SGX":"0x0000000000000000000000000000000000000000",
```

Comment:
why SGX address is 0x00?

Reply:
ohh because i just put dummy values, will change